### PR TITLE
Feature: Removed accent color from non-selected items in the column layout

### LINKED ISSUE #9594: Feature: Remove accent color from non-selected items in the column's layout
### Solution/Idea

image

### Requirements

- Remove accent color from non-selected items in the column's layout
- Non folders should still have the regular accented color section state

### Files Version

v2.3.3

### PR DESCRIPTION
**Pull request by marcelwgn**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #9594 

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
GIF of clicking around in Files with Column layout

### CODE REVIEW

**Comment by yaira2** [comment]
Is the width of the item supposed to change?

**Comment by chingucoding** [comment]
This was the only way I got this to work right now :(

**Comment by yaira2** [comment]
Is it possible to hide the selection indicator and build our own into the item template?

**Comment by chingucoding** [comment]
It would be possible but a lot of work I think

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs [hunk 1]
```diff
@@ -306,8 +327,26 @@ private async void FileList_SelectionChanged(object sender, SelectionChangedEven
...
+			if (e != null)
+			{
+				if (e.AddedItems.Count > 0)
+				{
+					columnsOwner?.HandleSelectionChange(this);
+				}
+
+				if (e.RemovedItems.Count > 0)
+				{
+					if (openedFolderPresenter != null)
```
Comment (QuaintMako) [inline src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs:340]: `If`s can be merged

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs [hunk 2]
```diff
@@ -408,7 +447,7 @@ private async void FileList_PreviewKeyDown(object sender, KeyRoutedEventArgs e)
 			}
 			else if (e.Key == VirtualKey.Right) // Right arrow: switch focus to next column
 			{
-				if (IsRenamingItem || (ParentShellPageInstance is not null && ParentShellPageInstance.ToolbarViewModel.IsEditModeEnabled)) 
+				if (IsRenamingItem || (ParentShellPageInstance is not null && ParentShellPageInstance.ToolbarViewModel.IsEditModeEnabled))
```
Comment (QuaintMako) [inline src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs:450]: Identifical to l.441, could be merged inside a function.

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs (hunk 1 above)
Comment (chingucoding) [reply src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs]: Good point, thanks!"

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs (hunk 2 above)
Comment (chingucoding) [reply src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs]: I think that would be a change more suitable for another PR.

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs [hunk 3]
```diff
@@ -33,12 +34,32 @@ public sealed partial class ColumnViewBase : BaseLayout
...
+		}
+
+		private void ClearOpenedFolderSelectionIndicator()
+		{
+			if (openedFolderPresenter != null)
+			{
+				openedFolderPresenter.Background = new SolidColorBrush(Microsoft.UI.Colors.Transparent);
+				var presenter = openedFolderPresenter.FindDescendant<Grid>()!;
+				presenter!.Background = new SolidColorBrush(Microsoft.UI.Colors.Transparent);
+			}
```
Comment (QuaintMako) [inline src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs:62]: I'd invert the condition in the `if` to get a `if(openedFolderPresenter is null) return` instead.

File: src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs [hunk 4]
```diff
@@ -306,8 +331,23 @@ private async void FileList_SelectionChanged(object sender, SelectionChangedEven
 
 			if (SelectedItems.Count == 1 && App.AppModel.IsQuickLookAvailable)
 				await QuickLookHelpers.ToggleQuickLook(ParentShellPageInstance, true);
+
+			if (e != null)
+			{
+				if (e.AddedItems.Count > 0)
+				{
+					columnsOwner?.HandleSelectionChange(this);
+				}
```
Comment (QuaintMako) [inline src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs:340]: Single instruction `if`s do not take curly brackets for us

File: src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs [hunk 5]
```diff
@@ -24,6 +24,18 @@ public ColumnViewBrowser() : base()
...
 
+		public void HandleSelectionChange(ColumnViewBase initiator)
+		{
+			foreach (var blade in ColumnHost.ActiveBlades)
+			{
+				var columnView = blade.FindDescendant<ColumnViewBase>();
+				if (columnView != null && columnView != initiator)
+				{
+					columnView.ClearSelectionIndicator();
+				}
```
Comment (QuaintMako) [inline src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs:35]: Single instruction `if`

**Review by QuaintMako — COMMENTED** [review]
Very small formatting issues, aside LGTM.

**Review by QuaintMako — DISMISSED** [review]

**Comment by yaira2** [comment]
@chingucoding is keyboard navigation still an issue?

**Comment by marcelwgn** [comment]
@yair100 It seems to work now, but I would prefer if you would double check.

**Review by yair100 — APPROVED** [review]
LGTM

**Review by QuaintMako — APPROVED** [review]

**Review by hecksmosis — APPROVED** [review]

**Pull request merged**